src/shared/mount-setup.c

```diff
@@ -148,6 +148,22 @@ bool mount_point_ignore(const char *path) {
         return false;
 }
 
+bool memory_recursiveprot_supported(void) {
+        int r;
+
+        r = mount_option_supported("cgroup2", "memory_recursiveprot", "yes");
+        if (r < 0) {
+                log_debug_errno(r, "Failed to determine whether 'memory_recursiveprot' option "
+                                    "is supported: %m");
+                return false;
+        }
+
+        if (r == 0)
+                log_debug("This version of kernel may not support 'memory_recursiveprot' - skipping it");
```

Comment:
may → does

"This kernel version does not support 'memory_recursiveprot', not using mount option.`

Reply:
fixed.